components/emoji_picker/modules/emoji_tabset.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="d-emoji-picker__tabset">
+    <dt-tab-group
+      tab-list-class="d-emoji-picker__tabset-list"
+      :selected="selectedTab"
+    >
+      <template #tabs>
+        <dt-tab
+          v-for="tab in tabs"
+          :id="tab.id"
+          :key="tab.id"
+          :panel-id="tab.panelId"
+          :label="tab.label"
+          @click.capture.stop="selectTabset(tab.id)"
+        >
```

Comment:
Can you override aria-controls to be your scrollable content element? That seems to be the remaining accessibility issue.

Reply:
Yay!, great, thanks, I didn't have a clue on what's even the problem lol.
Im guessing we sure put aria-controls required on dt-tab component or sth like that.